### ISSUE TITLE
Add new conversion blocks 'from_Hz' and 'to_Hz' to the MSL (from Modelica_LinearSystems)

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#156]
**Reported by dietmarw on 24 Mar 2009 10:43 UTC**
In the `Modelica_LinearSystems` library are two convenient conversion blocks that would be handy to have in the MSL:

```text/x-modelica
Modelica_LinearSystems.Internal.to_Hz
Modelica_LinearSystems.Internal.from_Hz
```

The place I would put them would be:
```text/x-modelica
Modelica.SIunits.Conversions.to_Hz
Modelica.SIunits.Conversions.from_Hz
```

So any chance of moving them there? The `Modelica_LineraSystems` could be updated accordingly to avoid duplicates (also one could take that opportunity to get rid of the dummy enumeration which is still used in that library and not necessary anymore).

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/156

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 21 Jun 2009 10:03 UTC**
Fixed in e3f1125508aaad02aaf20bebe7820ba0dd6fcf31

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 21 Jun 2009 10:08 UTC**
Fixed in e3f1125508aaad02aaf20bebe7820ba0dd6fcf31 and 6ea746c4165f9572ac7ada2d3b2f4532ce3b55c1
